MotMystere.cpp

```diff
@@ -2,19 +2,41 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+using namespace std;
+
+string getUserInput() {
+    string motMystere;
+    cout << "Entrer un mot : ";
+    cin >> motMystere;
+    cout << "Le mot est :"  << motMystere << "\n";
+    return motMystere;
+}
 
 int main()
 {
-    std::cout << "Hello World!\n";
+    string motMystere = getUserInput();
+    
+    //return 0;
+   
+    do
+    {
+        cout << "Quel est ce mot?" << endl;
+        cin >> motMystere;
+        cout << "Ce n'est pas le mot?" << endl;
+        cout << "Quel est ce mot?" << endl;
+        cin >> motMystere;
+
+    } while (motMystere !="Procra");
+
+        cout << "Bravo!" << endl;
+       // return 0;
```

Comment:
At the end of your function main, you should maintain the return 0, since your program should always return an int.

And if I'm not mistaken, it should not compile this way, does it compile?